Source/SwiftLintFramework/Rules/MarkRule.swift

```diff
@@ -35,10 +35,7 @@ public struct MarkRule: ConfigurationProviderRule {
         let options = ["MARK:[^ ]", "[^ ]MARK: [^-]", "\\sMARK:[^ ]", "MARK:[ ][-][^ ]"]
         let pattern = "(" + options.joinWithSeparator("|") + ")"
 
-        return file.matchPattern(pattern).flatMap { range, syntaxKinds in
```

Comment:
`isCommentLike` _is_ wider, because it also includes doc comments, but `// MARK` comments can't be doc comments, they must be double-slashed. Whereas `TODO` comments could be left in documentation comments.